### ISSUE TITLE
one login page many apps

### ISSUE THREAD

**Issue by fmarois** [datastorm-open/shinymanager#31]
Hello I am currently using shiny-server open source with nginx login and I would like to use shinymanager to replace that. More precisely I wanted to know if it was possible to have one login being done with shinymanager which then provide access to many other apps, located in the same server but not in the same app. Maybe using the token generated on one app and transferring it to another app but I could not achieve this. 
By the way your package is amazing keep up the good work.
Merci

**Comment by pvictor** [comment]
Hello,

Thanks for your message!
That's not possible right now, but we could extend the package to do this.

Victor

**Comment by deann88** [comment]
Thanks for the package! I agree this would be helpful.

**Comment by FGG-Belfast** [comment]
I also agree this would be a useful feature. Would love to be able to have an app that sits in the parent directory and then allows access to all apps within.
